viper-runner/benchmark.conf

```diff
@@ -0,0 +1,64 @@
+## Viper runner configuration file in HOCON format,
+## to be used for benchmarking in our CPSC 513 project
+## on inlined predicates.
+
+## Folder that contains the tests. Only .sil files will be considered.
+## Alison: .vpr files work as well.
+test_folder = "./scripts/"
```

Comment:
`./scripts/` is where we can put all the `.vpr` files we want to evaluate the performance of silicon on.